api-tests/wallet_payment_api.tests.json

```diff
@@ -0,0 +1,1519 @@
+				},
+				{
+					"name": "Invalid input id",
+					"originalRequest": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "application/json"
+							},
+							{
+								"description": "Added as a part of security scheme: bearer",
+								"key": "Authorization",
+								"value": "Bearer <token>"
+							}
+						],
+						"url": {
+							"raw": "{{baseUrl}}/wallets/:walletId",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"wallets",
+								":walletId"
+							],
+							"variable": [
+								{
+									"key": "walletId"
+								}
+							]
+						}
+					},
+					"status": "Bad Request",
+					"code": 400,
+					"_postman_previewlanguage": "json",
+					"header": [
+						{
+							"key": "Content-Type",
+							"value": "application/json"
+						}
+					],
+					"cookie": [],
+					"body": "{\n  \"type\": \"about:blank\",\n  \"title\": \"<string>\",\n  \"status\": \"<integer>\",\n  \"detail\": \"<string>\",\n  \"instance\": \"<uri>\"\n}"
+				},
+				{
+					"name": "Wallet not found",
+					"originalRequest": {
+						"method": "GET",
+						"header": [
+							{
+								"description": "Added as a part of security scheme: bearer",
+								"key": "Authorization",
+								"value": "Bearer <token>"
+							}
+						],
+						"url": {
+							"raw": "{{baseUrl}}/wallets/:walletId",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"wallets",
+								":walletId"
+							],
+							"variable": [
+								{
+									"key": "walletId"
+								}
+							]
+						}
+					},
+					"status": "Not Found",
+					"code": 404,
+					"_postman_previewlanguage": "text",
+					"header": [],
+					"cookie": [],
+					"body": ""
+				},
+				{
+					"name": "Timeout serving request",
+					"originalRequest": {
+						"method": "GET",
+						"header": [
+							{
+								"description": "Added as a part of security scheme: bearer",
+								"key": "Authorization",
+								"value": "Bearer <token>"
+							}
+						],
+						"url": {
+							"raw": "{{baseUrl}}/wallets/:walletId",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"wallets",
+								":walletId"
+							],
+							"variable": [
+								{
+									"key": "walletId"
+								}
+							]
+						}
+					},
+					"status": "Gateway Timeout",
+					"code": 504,
+					"_postman_previewlanguage": "text",
+					"header": [],
+					"cookie": [],
+					"body": ""
+				}
+			]
+		},
+		{
+			"name": "Get wallet by user identifier",
```

Comment:
Add test assertions for get wallet by user identifier against expected status code and response body